README.md

```diff
@@ -1,2 +1,35 @@
-# Tic-Tac-Toe
-A game built with Ruby
+## Tic-Tac-Toe
+
+In this project we had the opportunity to build Tic Tac Toe game in Ruby. Just in case you're unfamiliar with the game, let's discuss the rules first. Tic Tac Toe is played on a 3 x 3 grid of available cells, and the goal of the game is to place three of your tokens (X or O) on the grid in a row, including diagonals.
+
+Tic Tac Toe is a game that flexes the basic concepts in programming. For example, a Tic Tac Toe program requires data structures for storing the board and conditional logic for knowing whose turn it is or if someone has won.
+
+It is a game played by two people, each trying to make a line of three X's or three O's in a boxlike figure with nine spaces. It is either won horizontally, vertically or diagonally.
+
+The first player that has a line of three X's or three O's wins the game.
```

Comment:
- divide this into two sections. move some content from here to a "Rules section" for the game.